app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        result = self.appetite if self.is_hungry else 0
+        if self.is_hungry:
```

Comment:
To avoid duplicating conditions and creating a new value, you can remove ternary operator and put this logic inside of the `if self.is_hungry` condition